api/src/utils/github.js

```diff
@@ -58,11 +58,37 @@ export const getUserByName = async (username) => {
   log.debug(`getUserByName ${username}`)
   const installations = await getAuthenticatedApps()
   const config = getConfig()
-
   const response = await installations.apps[
-    config.primaryOrg
+    config.primaryOrg.toLowerCase()
   ].authenticatedRequest('GET /users/{username}', {
     username,
   })
   return response.data
 }
+
+export const getRequestStatuses = async (requests) => {
+  log.info(`getRequestStatuses`)
+  let pendingUserRequests = []
+  const installations = await getAuthenticatedApps()
+
+  const config = getConfig()
+
+  for (const org of config.orgs) {
+    log.debug(`org being searched ${org}`)
+    const orgPendingRequests = await installations.apps[
+      org.toLowerCase()
+    ].authenticatedRequest('GET /orgs/{org}/invitations', {
+      org,
+    })
+    pendingUserRequests = pendingUserRequests.concat(filterOrgRequestsByUser(requests, orgPendingRequests, org))
+  }
```

Comment:
this routine is still a triple nested for loop. We need to take a look at this and see if theres a way to remove the amount of nested loops. This is not so much an optimization but a clarity task. it will be easier to read/debug